### PR TITLE
Update distinct-id-limits.md to be hot-shard-limits.md

### PR DESCRIPTION
**Pull request by Uduchi2nd**

### CODE REVIEW

**Review by mavlee — APPROVED** [review]

File: pages/docs/tracking-best-practices/distinct-id-limits.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
-# Distinct ID Limits
+# Hot Shard Limits
 
-In order to maintain fast queries and catch implementation mistakes, we set a limit on the number of events sent to a particular `distinct_id` in a given time window. This threshold has been established as **200K events per `distinct_id` per event date in the project**.
+In order to maintain fast queries and catch implementation mistakes, we set a limit on the number of events sent to a particular identifier in a given time window. This threshold has been established for a project as:
+- 200K events per `distinct_id` per event date in the project's user analytics.
```
Comment (kurbycchua) [inline pages/docs/tracking-best-practices/distinct-id-limits.md:4]: should we say 
- 200K events per `distinct_id` per event date in the project's user analytics (i.e without [Group Analytics](/docs/data-structure/advanced/group-analytics)).

File: pages/docs/tracking-best-practices/distinct-id-limits.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+- 1M events per `group identifier` per event date (still represented as distinct_id in the rest of this doc) in each of the project's [analytic groups](/docs/data-structure/advanced/group-analytics).
```
Comment (kurbycchua) [inline pages/docs/tracking-best-practices/distinct-id-limits.md:5]: should we say
- 1M events per `group identifier` per event date (still represented as `distinct_id` in the rest of this doc) in each of the project's [Group Key](/docs/data-structure/advanced/group-analytics#group-keys-in-project-settings).

File: pages/docs/tracking-best-practices/distinct-id-limits.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 
 ## What is a hot shard?
 Whenever a project goes above the threshold described above, it generates an imbalance when storing events across distinct_ids, where one distinct_id's events grows larger than the rest, impacting storage and query systems which in-turn results in high query latencies (slower reports) for the end user.
 
 Since we distribute events across shards, this imbalance is called a **hot shard**.
 
+Group analytics has a different storage sharding of events separate from the user analytics. A hot shard that appears in an analytic group may or may not not appear in user analytics.
```
Comment (kurbycchua) [inline pages/docs/tracking-best-practices/distinct-id-limits.md:12]: should we say 

Group analytics has a different storage sharding of events separate from the user analytics. A hot shard that appears for a `group_identifier` in a Group Key may or may not not appear as a hot shard in user analytics level.

**Review by kurbycchua — COMMENTED** [review]
left some comments, maybe @argenisf can also take a look

**Pull request merged**